scripts/ci-entrypoint.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
i think a ci entrypoint will the good to have to make a standard entry point for e2e/conformance tests.

on those scripts we have some other things that are in common like setup the gcp network and build the node image if needed. maybe we can add those here and remove the duplication from those scripts. wdyt?
then those scripts should inkove this in the start to setup all the requirements.

Reply:
Hi, @cpanato will it be okay if you can explain me a bit more in a detailed way? this is my first time writing a script, so what things should be added if you could explain me more clearly, it will be helpful

Reply:
It will be easier for me if I get an example of what things to use for the following